Godeps/Godeps.json

```diff
@@ -198,6 +198,11 @@
 			"Rev": "535138d7bcd717d6531c701ef5933d98b1866257"
 		},
 		{
+			"ImportPath": "github.com/go-ini/ini",
```

Comment:
@dmcgowan We have removed "github.com/go-ini/ini" from Godeps.json . But ci/circleci build is failing stating "package github.com/go-ini/ini not found"